application/comit_node/Cargo.toml

```diff
@@ -18,30 +18,24 @@ directories = "1.0"
 either = "1.5"
 enum-display-derive = "0.1"
 failure = "0.1"
-frunk = "0.2"
```

Comment:
❓ is there no way to automatise this?

Reply:
Probably a cargo plugin, a quick search didn't bring anything up :(